cmd/monaco/integrationtest/v2/test-resources/settings-unique-properties/project1/builtinmonitoring.slo/d955d9b7-9630-3be5-aca5-3715779a7282.json

```diff
@@ -0,0 +1,15 @@
+{
+  "enabled": true,
+  "name": "Just a Name",
+  "metricName": "justaname",
```

Comment:
From the failing message:
`Metric name field can not start with a number, contain special characters or uppercase letters.`

Reply:
looking at the other SLO config it does not look like the metric name is modified? 
cmd/monaco/integrationtest/v2/test-resources/settings-unique-properties/project2/builtinmonitoring.slo/d955d9b7-9630-3be5-aca5-3715779a7282.json

still has `"metricName": "justaname",` - but changes the `name`

Reply:
The second comment is for documentation. `metricName` value was the only problem.